src/helpers/cauldron/getInterest.test.js

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect } from "vitest";
+
+import { BigNumber } from "ethers";
+
+import { getInterest } from "./getInterest";
+
+describe("getInterest()", () => {
+  it("should return interest if accrueInfo INTEREST_PER_SECOND more than 0", () => {
+    const accrueInfo = {
+      INTEREST_PER_SECOND: BigNumber.from(1e8),
+    }
+    const result = getInterest(accrueInfo);
+    expect(Number(result)).toBeGreaterThan(0);
```

Comment:
Probably worth using a toBe here so that people understand whether the output should have decimals or not... e.g., If I'm expecting the output to be 3% (just as an example), when I look at these tests, I don't understand whether the function should return 3, 0.03, or 30000000.

Reply:
Also, does this even pass? I see that in the TypeScript file, you're returning a string...

Reply:
updated